components/SideBarInfoIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react'
+import { Flex, useColorMode } from '@chakra-ui/react'
+import { NAMED_COLORS } from 'theme/constants'
+
+interface SideBarInfoIconProps {
+  label: string
+}
+
+const SideBarInfoIcon: FC<SideBarInfoIconProps> = ({ label }) => {
+  const { colorMode } = useColorMode()
+
+  return (
+    <Flex
+      id="SidebarInfoIcon"
+      w={'1.438rem'}
+      h={'1.5rem'}
+      flexDir={'column'}
+      justify={'center'}
+      align={'center'}
+      p="0.063rem 0.375rem"
+      borderRadius="0.125rem"
+      bgColor={
+        colorMode === 'light'
+          ? NAMED_COLORS.LIGHT_GREY
+          : NAMED_COLORS.DARKER_GREY
+      }
```

Comment:
For me some of this properties must be in `theme\components\[componentName].ts`.
Also this component cannot be customized via properties. For example `height`, `width`.